src/avgPerceptronTagger_DataDeps.jl

```diff
@@ -0,0 +1,14 @@
+using DataDeps
```

Comment:
add this to the averagePerceptronTagger.jl file? Seems very little code for a new file.

Reply:
Done.